### PR TITLE
Fix /datum/weakref appearing when linking airlock heretic portals

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

Fix /datum/weakref appearing when linking airlock heretic portals

## Why It's Good For The Game

buggo

impropero

## Changelog

:cl:
fix: Fix /datum/weakref appearing when linking airlock heretic portals
/:cl:

### CODE REVIEW

File: code/modules/antagonists/heretic/items/keyring.dm [hunk 1]
```diff
@@ -185,7 +185,7 @@
 
 	if(reference_resolved)
 		make_portal(user, reference_resolved, target)
-		to_chat(user, span_notice("You use [src], to link [link] and [target] together."))
+		to_chat(user, span_notice("You use [src], to link \the [reference_resolved] and \the [target] together."))
```
Comment (MrMelbert) [inline code/modules/antagonists/heretic/items/keyring.dm:188]: For the record the `\the` helpers here are doing absolutely nothing, `\the` is added regardless when you stringify something under most circumstances

Comment (carlarctg) [reply code/modules/antagonists/heretic/items/keyring.dm:188]: The airlocks in runtime station weren't adding it

Comment (MrMelbert) [reply code/modules/antagonists/heretic/items/keyring.dm:188]: It adds it if necessary, so in this case it did not deem it necessary

Comment (carlarctg) [reply code/modules/antagonists/heretic/items/keyring.dm:188]:
```suggestion
		to_chat(user, span_notice("You use [src], to link [reference_resolved] and [target] together."))
```

**Pull request merged**